content/zh/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md

```diff
@@ -0,0 +1,244 @@
+```
+
+<!--
+`advertiseAddress` in InitConfiguration specifies the IP address that the API Server will advertise it is listening on. The value of `advertiseAddress` equals the `--apiserver-advertise-address` flag of `kubeadm init`
+
+Run kubeadm to initiate the dual-stack control plane node:
+-->
+InitConfiguration 中的 `advertiseAddress` 给出 API 服务器将公告自身要监听的
+IP 地址。`advertiseAddress` 的取值与 `kubeadm init` 的标志
+`--apiserver-advertise-address` 的取值相同。
+
+运行 kubeadm 来实例化双协议栈控制面节点：
+
+```shell
+kubeadm init --config=kubeadm-config.yaml
+```
+
+<!--
+The kube-controller-manager flags `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6` are set with default values. See [configure IPv4/IPv6 dual stack](/docs/concepts/services-networking/dual-stack#configure-ipv4-ipv6-dual-stack).
+-->
+kube-controller-manager 标志 `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6`
+是使用默认值来设置的。参见[配置 IPv4/IPv6 双协议栈](/zh/docs/concepts/services-networking/dual-stack#configure-ipv4-ipv6-dual-stack)。
+
+{{< note >}}
+<!--
+The `--apiserver-advertise-address` flag does not support dual-stack.
+-->
+标志 `--apiserver-advertise-address` 不支持双协议栈。
+{{< /note >}}
+
+<!--
+### Join a node to dual-stack cluster
+
+Before joining a node, make sure that the node has IPv6 routable network interface and allows IPv6 forwarding.
+
+Here is an example kubeadm [configuration file](https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3) `kubeadm-config.yaml` for joining a worker node to the cluster.
+-->
+### 向双协议栈集群添加节点   {#join-a-node-to-dual-stack-cluster}
+
+在添加节点之前，请确保该节点具有 IPv6 可路由的网络接口并且启用了 IPv6 转发。
+
+下面的名为 `kubeadm-config.yaml` 的 kubeadm
+[配置文件](/docs/reference/config-api/kubeadm-config.v1beta3/)
+示例用于向集群中添加工作节点。
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: JoinConfiguration
+discovery:
+  bootstrapToken:
+    apiServerEndpoint: 10.100.0.1:6443
+    token: "clvldh.vjjwg16ucnhp94qr"
+    caCertHashes:
+    - "sha256:a4863cde706cfc580a439f842cc65d5ef112b7b2be31628513a9881cf0d9fe0e"
+    # change auth info above to match the actual token and CA certificate hash for your cluster
+nodeRegistration:
+  kubeletExtraArgs:
+    node-ip: 10.100.0.3,fd00:1:2:3::3
+```
+
+<!--
+Also, here is an example kubeadm [configuration file](https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3) `kubeadm-config.yaml` for joining another control plane node to the cluster.
+-->
+下面的名为 `kubeadm-config.yaml` 的 kubeadm
+[配置文件](/docs/reference/config-api/kubeadm-config.v1beta3/)
+示例用于向集群中添加另一个控制面节点。
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: JoinConfiguration
+controlPlane:
+  localAPIEndpoint:
+    advertiseAddress: "10.100.0.2"
+    bindPort: 6443
+discovery:
+  bootstrapToken:
+    apiServerEndpoint: 10.100.0.1:6443
+    token: "clvldh.vjjwg16ucnhp94qr"
+    caCertHashes:
+    - "sha256:a4863cde706cfc580a439f842cc65d5ef112b7b2be31628513a9881cf0d9fe0e"
+    # change auth info above to match the actual token and CA certificate hash for your cluster
```

Comment:
nit: you could localize this comment too

Reply:
Thanks.